README.md

```diff
@@ -51,10 +51,42 @@ Then, in your `.eslintrc.json`:
 }
 ```
 
+## Recommended Configuration
+
+This plugin works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect patterns in the AST. (If another active ESLint rule disagrees with `prettier` about how code should be formatted, it will be impossible to avoid lint errors.) You can use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all formatting-related ESLint rules.
+
+To integrate this plugin with `eslint-config-prettier`, you can use the `"recommended"` configuration.
+
+Install `eslint-config-prettier` as well:
+
+```sh
+npm install --save-dev eslint-config-prettier
+```
+
+Then all you need in your `.eslintrc.json` is:
+
+```json
+{
+  "extends": [
+    "plugin:prettier/recommended"
+  ]
```

Comment:
Is it also necessary to use `"plugins": ["prettier"]` in this config file?

Reply:
No, configurations can turn on plugins, including their own.

Reply:
Hmm, I had been under the impression that `"plugins": ["prettier"]` was needed to use `plugin:prettier/recommended` at all, but after trying it out it seems like you're correct.

Separately, maybe it would be a good idea to update ESLint's plugin-loading so that `extends: plugin:foo/bar` also implies `plugins: ["foo"]`, so that it wouldn't be necessary to do something like `plugins: ["prettier"]` in a shareable config within `eslint-plugin-prettier` itself.

Reply:
Yeah I was surprised that it works the way it does. Not sure if there's a use case for a plugin exporting a configuration without exposing its rules.